packages/components/src/range-control/index.js

```diff
@@ -25,6 +19,8 @@ import Button from '../button';
 import Dashicon from '../dashicon';
 
 import { color } from '../utils/colors';
+
```

Comment:
```suggestion
```

(We shouldn't want or need the extra lines amongst these imports, unless there's rhyme or reason to it)